src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -2805,6 +2818,17 @@ public static boolean isAutoSnapshot()
         return conf.auto_snapshot;
     }
 
+    public static DurationSpec getAutoSnapshotTtl()
+    {
+        return autoSnapshoTtl;
+    }
+
+    @VisibleForTesting
+    public static void setAutoSnapshotTtl(DurationSpec autoSnapshotTtl)
+    {
+        autoSnapshoTtl = autoSnapshotTtl;
```

Comment:
this is strage at first sight, you are basically assigning variable to static variable of the same name, can not be this somehow changed?